dali/util/local_file.cc

```diff
@@ -36,18 +38,18 @@
 static int _sysctl(struct __sysctl_args *args);
 
 static int get_max_vm_cnt() {
-  struct __sysctl_args args = {0, };
-  int vm_cnt;
+  int vm_cnt = 1;
+#if !defined(__AARCH64_GNU__)
```

Comment:
Doesn't AARCH64 have `/proc/sys/vm/max_map_count`? Maybe we can fallback to L54 immediately for it?

Reply:
It does, I can change it.

Reply:
Done